### DISCUSSION TITLE
vercel doesnt load swagger css file

### DISCUSSION THREAD

**Discussion by m0x61h0x64i** [vercel/vercel#8631] [Help] [answered]
when i tried to use swagger on my vercel hobby account, after deployment my api documentation doesnt load currectly.

the css file is missing.

but it works well on localhost.

**Comment by amyegan** [comment, marked as answer]
It looks like this is a duplicate of https://github.com/vercel/vercel/discussions/8630. Locking this discussion in favor of consolidating conversation and info there.
